pkg/apparchive/helm-v1beta2.go

```diff
@@ -159,16 +162,13 @@ func WriteV1Beta2HelmCharts(opts WriteV1Beta2HelmChartsOptions) error {
 			return errors.Wrap(err, "failed to process online images")
 		}
 
-		upstreamDir := opts.Upstream.GetUpstreamDir(opts.WriteUpstreamOptions)
```

Comment:
I think [this](https://github.com/replicatedhq/kots/pull/4239/files#diff-9bd889d7bad0a647ad15e3502d8a58d94cef268bbde15e348c874b4c770cb3fbR594) is the reason why we reloaded the installation file here? Want to make sure that this would not overwrite other changes that happened along the way.